src/main/java/net/sf/jabref/gui/entryeditor/FieldExtraComponents.java

```diff
@@ -474,4 +474,55 @@ public void mouseClicked(MouseEvent e) {
         return Optional.of(gender);
 
     }
+
+    /**
+     * Return a button which sets the owner if the field for fields with EXTRA_SET_OWNER
```

Comment:
Yes, a bit of copy paste coding, like checkURL...

I'll sort it out. "Go to parent" may be a better name, but I wasn't sure
how well defined "parent" actually is here and somehow what happens is that
the parent entry is selected. And couldn't figure out a good translation to
Swedish...

Another issue is that the button is enabled when the entry editor is
enabled. At least for new entries... Once something is typed it works
though. Guess one should try to trigger the documentListener once, but
didn't figure out how...